JavaDirectory/src/main/java/com/bug/free/invention/api/Services/JobService.java

```diff
@@ -0,0 +1,48 @@
+package com.bug.free.invention.api.Services;
+
+import com.bug.free.invention.api.Models.Job;
+import com.bug.free.invention.api.controllers.DBConfig;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
+
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class JobService {
+    //Load From Database
+    @Autowired
+    private JobRepository repository;
+
+    public void populateJobRoles() {
+        try {
+            List<Job> Jobs = new ArrayList<>();
+            List<Job> savedJobs = new ArrayList<>();
+            Statement statement = DBConfig.getConnection().createStatement();
+            //'Job_ID','Capability_ID','Band_ID','Job_Title'
+            String dbQuery = "SELECT * FROM `Job` INNER JOIN(Capability,Band,JobSummary) ON (Job.Band_ID = Band.Band_ID) and (Job.Capability_ID = Capability.Capability_ID) and (JobSummary.Job_ID = Job.Job_ID) ";//"GROUP BY 'Capability_ID','Band_ID','Job_Title'";
+            ResultSet results = statement.executeQuery(dbQuery);
+            while (results.next()) {
```

Comment:
```suggestion
            if (results.next()) {
```
Slightly cleaner execution